src/OpenTelemetry.Abstractions/Trace/SpanData.cs

```diff
@@ -148,7 +148,7 @@ public class SpanData
         {
             if (events == null)
             {
-                events = TimedEvents<IEvent>.Create(new List<ITimedEvent<IEvent>>(), 0);
+                events = TimedEvents<IEvent>.Create(Enumerable.Empty<ITimedEvent<IEvent>>(), 0);
```

Comment:
I've recently added TimedEvents.Empty (https://github.com/open-telemetry/opentelemetry-dotnet/blob/4952219a8eb443582c0985021135c1121d92ff83/src/OpenTelemetry.Abstractions/Trace/Export/TimedEvents.cs#L25) please use it here

Reply:
The downside of this approach, even if made `internal` and used is that it'll create an instance for each type of `T` which kind of defeats the purpose of the `Empty` field IMHO.
Let me know if you still want me to turn this into an `internal` field and go ahead with it regardless.